### PR TITLE
Solo teardown

### PR DESCRIPTION
**Pull request by tahiry-dev**
Send pull request for my teardown project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,393 @@
...
+    <header>
+        <div class="nav-box-logo most-pleasant border-radius"></div>
+        <div class="nav-boxes"></div>
+        <div class="nav-boxes"></div>
+        <div class="nav-boxes"></div>
+        <div class="nav-boxes"></div>
+        <div class="nav-boxes"></div>
+        <div class="nav-box-more most-pleasant border-radius"></div>
+        <div class="nav-box-search most-pleasant border-radius"></div>
+    </header>
```
Comment (SpaYco) [inline index.html]: - [ ] [Optional] use `nav` element with `ul`/`li` inside

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) (add something like the steps section in your README.md)

- [ ] you have repetitive code in your CSS, decrease it by making one class for a particular property (eg. `.d-flex` for `display: flex;`) so you won't have to repeat it again


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by tahiry-dev** [comment]
- I used Toggle Pesticide to help me divide the whole page into boxes.
- ”Translated” these boxes into empty HTML Containers
- Aligned these containers properly using Grid, Flexbox, and Float
- Gave general color to all of the boxes
- Reviewed the design principles
- Gave darker color to the most pleasant area of boxes

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

P.S: you can edit the very first post in your PR instead of posting a comment, please next time describes the project on the first post of the PR.

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**